_troubleshooting/self-hosted/pg_dump-permission-denied.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Permission denied for table job_errors when running `pg_dump`
```

Comment:
job_errors shows up as job _errors_ in the document. Can you add the backticks to job_errors as well?

Reply:
done!!!